src/pages/index.js

```diff
@@ -1,30 +1,39 @@
 import "./index.css";
-import { config, formValidators, initialCards } from "../utils/constants.js";
+import { config, formValidators } from "../utils/constants.js";
+import Api from "../components/Api.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this._disableButton(this._submitButton, this._settings);` from line 62 in `FormValidator.js` because you need to disable the button only in block `then` after resetting the form